charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook-with-cert-manager.yaml

```diff
@@ -18,6 +18,14 @@ webhooks:
         path: /mutate-opentelemetry-io-v1alpha1-instrumentation
     failurePolicy: Fail
     name: minstrumentation.kb.io
+    {{- if .Values.admissionWebhooks.namespaceSelector }}
```

Comment:
Does the other webhook file need the same changes?

Reply:
No, That's fine.